README.md

```diff
@@ -120,19 +120,27 @@ _As far as I know, there's nothing we can do about those two limitations. Please
 But you generally only use stubs during the development phase and want to remove your stubs when submitting to the AppStore. So be careful to only include `OHHTTPStubs` in your test targets, or only use it in `#if DEBUG` portions, to avoid forgetting to remove it when the time comes that you release for the AppStore!
 
 
-## About OHHTTPStubs own's Unit Tests
+# Building OHHTTPStubs
 
-If you want to be able to run `OHHTTPStubs`' Unit Tests, be sure you cloned the [`AFNetworking`](https://github.com/AFNetworking/AFNetworking/) submodule (by using the `--recursive` option when cloning your repo, or using `git submodule init` and `git submodule update`) as it is used by some of `OHHTTPStubs` unit tests.
+## Building OHHTTPStubs
+To build (and run unit tests) `OHHTTPStubs`, use cocoapods. We are using [`AFNetworking`](https://github.com/AFNetworking/AFNetworking/) for unit tests only. No other dependencies are used for `OHHTTPStubs`.
```

Comment:
This is only useful if one want to build and run the Unit tests.

Anyone only wanting to build the library or dynamic framework shouldn't need to run `pod install`.
Needing to do so probably means that it will break the `Carthage` support that we just got some days ago anyway.